docs/notes/tutorial_index.rst

```diff
@@ -0,0 +1,52 @@
+.. _tutorial_index:
+
+Tutorial Index
+==============
+
+Kaolin provides tutorials as ipython notebooks, docs pages and simple scripts. Note that the links
+point to master.
+
+
+Detailed Tutorials
+------------------
+
+* `Deep Marching Tetrahedra <https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/dmtet_tutorial.ipynb>`_: reconstructs a tetrahedral mesh from point clouds with `DMTet <https://nv-tlabs.github.io/DMTet/>`_, covering:
+    * generating data with Omniverse Kaolin App
+    * loading point clouds from a ``.usd`` file
+    * chamfer distance as a loss function
+    * differentiable marching tetrahedra
+    * using Timelapse API for 3D checkpoints
+    * visualizing 3D results of training
+* `Understanding Structured Point Clouds (SPCs) <https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/understanding_spcs_tutorial.ipynb>`_: walks through SPC features, covering:
+    * under-the-hood explanation of SPC, why it's useful and key ops
+    * loading a mesh
+    * sampling a point cloud
+    * converting a point cloud to SPC
+    * setting up camera
+    * rendering SPC with ray tracing
+    * storing features in an SPC
+* `Differentiable Rendering <https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/dibr_tutorial.ipynb>`_: optimizes a triangular mesh from images using `DIB-R <https://github.com/nv-tlabs/DIB-R-Single-Image-3D-Reconstruction>`_ renderer, covering:
```

Comment:
Same question than for spc but for `dibr_tutorial.ipynb` and https://kaolin--534.org.readthedocs.build/en/534/notes/diff_render.html

Reply:
Same answer :)